code/modules/clothing/spacesuits/syndi.dm

```diff
@@ -66,3 +66,38 @@
 		rad = 25
 	)
 	siemens_coefficient = 0.6
+
+//Upgraded version of the above, found rarely in greyson loot.
```

Comment:
were does this spawn to be rarely in GP loot?

Reply:
It was added to the list of greyson clothing spawns with weight 1 - compared to the weight 8 of the regular space suit